File: src/inventory/ClientApp/app/shared/model/service/service.model.js
System.register([], function (exports_1, context_1) {
    "use strict";
    var ServiceModel;
    var __moduleName = context_1 && context_1.id;
    return {
        setters: [],
        execute: function () {
            ServiceModel = /** @class */ (function () {
                function ServiceModel(name, displayName, startMode, state) {
                    this.name = name && name || "";
                    this.displayName = displayName && displayName || "";
                    this.startMode = startMode && startMode || "";
                    this.state = state && state || "";
                }
                return ServiceModel;
            }());
            exports_1("ServiceModel", ServiceModel);
        }
    };
});

File: src/inventory/Models/ServiceModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace inventory.Models
{
    public class ServiceModel
    {
        public string Name { get; set; }
        public string DisplayName { get; set; }
        public string StartMode { get; set; }
        public string State { get; set; }
    }
}
